sdk/confidentialledger/azure-security-confidentialledger/pom.xml

```diff
@@ -104,4 +104,4 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
-</project>
+</project>
```

Comment:
previous change to this pom.xml is correct. why revert?

Reply:
@deer-prudence I think your editor is removing the newlines here. That's what the red circle with the dash is warning about. It looks like both pom files, tests.yml, test-resource.json and several of the java files have this issue.

Reply:
I see; thank you for explaining!